.changeset/tiny-pandas-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/gatsby-theme-docs': patch
+---
+
+Fix css bug in search dialog.
```

Comment:
Nit: can we provide a bit more information about this fix? For instance, how does "moving an ID" fixes a CSS bug? It's not really clear...

Thanks

Reply:
I explained it in the PR description. But I can change the changelog text to something more specific if you want.

Reply:
Yeah in general I would prefer to also have the changesets with good enough information.

For instance, reading this "Fix css bug in search dialog." I would think it's a fix in the css styles (colors, borders, etc.) but it's actually 1) not a change in css styles and 2) something totally unrelated to css.